### PR TITLE
Feature/yoojin/withdraw complete

### PR DESCRIPTION
**Pull request by yoojinko**
Add new method (withdrawComplete) and type (WithdrawRequestList)

### CODE REVIEW

File: src/asset.ts [hunk 1]
```diff
@@ -208,4 +208,15 @@ export default class Asset extends AinftBase {
...
     return this.sendRequest(HttpMethod.GET, trailingUrl, query);
   }
+
+  withdrawComplete(
+    appId: string,
+    symbol: string,
+    requestList: WithdrawRequestList,
+    txHash: string,
+  ): Promise<void> {
+    const body = { appId, requestList, txHash};
```
Comment (woomurf) [inline src/asset.ts:218]: minor) `txHash }` space?

**Review by woomurf — APPROVED** [review]
LGTM with minor comments. Thanks!

File: src/asset.ts (hunk 1 above)
Comment (yoojinko) [reply src/asset.ts]: Oh gosh, Thanks.

**Comment by yoojinko** [comment]
Thanks for the review!

**Pull request merged**